lib/beacon_web/components/components.ex

```diff
@@ -32,6 +32,26 @@ defmodule BeaconWeb.Components do
     """
   end
 
+  @doc """
+  Renders a the amount in minutes for a page to be read.
+
+  ## Examples
+
+      <BeaconWeb.Components.reading_time />
+  """
+
+  def reading_time(assigns) do
+    %{template: content} = Beacon.Content.get_page_by(Process.get(:__beacon_site__), path: Process.get(:__beacon_page_path__))
+    content_without_html_tags = String.replace(content, ~r/(<[^>]*>|\n|\s{2,})/, "", global: true)
+    words_per_minute = 270
+    estimated_time_in_minutes = Kernel.trunc((String.split(content_without_html_tags, " ") |> length()) / words_per_minute)
+    assigns = Map.put(assigns, :estimated_time_in_minutes, estimated_time_in_minutes)
+
+    ~H"""
+    <%= @estimated_time_in_minutes %> min read
```

Comment:
```suggestion
    <%= @estimated_time_in_minutes %>
```

It can return just the time (the number) so the user has the freedom to format and style as needed

Reply:
https://github.com/BeaconCMS/beacon/pull/388/commits/2af97cf996cb462c11e5dfbac9f6f32400375653